ghost/i18n/locales/sl/portal.json

```diff
@@ -1,212 +1,212 @@
     "Sending login link...": "Pošiljanje povezave za prijavo...",
     "Sending...": "Pošiljanje...",
-    "Show all": "",
+    "Show all": "Pokaži vse",
     "Sign in": "Prijava",
-    "Sign out": "",
+    "Sign out": "Odjava",
     "Sign up": "Registracija",
-    "Signup error: Invalid link": "",
-    "Something went wrong, please try again later.": "",
-    "Sorry, no recommendations are available right now.": "",
-    "Sorry, that didn’t work.": "",
-    "Spam complaints": "",
-    "Start {{amount}}-day free trial": "Začetek {{amount}}-dnevnega brezplačnega preizkusa",
-    "Starting {{startDate}}": "",
-    "Starting today": "",
-    "Submit feedback": "Pošljite povratno informacijo",
-    "Subscribe": "",
-    "Subscribed": "",
-    "Subscription plan updated successfully": "",
-    "Success": "",
-    "Success! Check your email for magic link to sign-in.": "",
-    "Success! Your account is fully activated, you now have access to all content.": "",
-    "Success! Your email is updated.": "",
+    "Signup error: Invalid link": "Napaka pri registraciji: Neveljavna povezava",
+    "Something went wrong, please try again later.": "Nekaj je šlo narobe, poskusite ponovno kasneje.",
+    "Sorry, no recommendations are available right now.": "Oprostite, trenutno ni priporočil.",
+    "Sorry, that didn’t work.": "Oprostite, to ni uspelo.",
+    "Spam complaints": "Pritožbe zaradi vsiljene pošte",
+    "Start {{amount}}-day free trial": "Začnite {{amount}}-dnevni brezplačni preizkus",
+    "Starting {{startDate}}": "Začetek {{startDate}}",
+    "Starting today": "Z današnjim dnem",
+    "Submit feedback": "Pošljite povratne informacije",
+    "Subscribe": "Naročite se",
+    "Subscribed": "Naročen",
+    "Subscription plan updated successfully": "Načrt naročnine je bil uspešno posodobljen",
+    "Success": "Uspeh",
+    "Success! Check your email for magic link to sign-in.": "Uspešno! Preverite svojo e-pošto za čarobno povezavo za prijavo.",
+    "Success! Your account is fully activated, you now have access to all content.": "Uspešno! Vaš račun je popolnoma aktiviran, zdaj imate dostop do vseh vsebin.",
+    "Success! Your email is updated.": "Uspešno! Vaš e-poštni naslov je posodobljen.",
     "Successfully unsubscribed": "Uspešno ste se odjavili",
-    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "",
-    "Thank you for your support": "",
-    "Thank you for your support!": "",
+    "Thank you for subscribing. Before you start reading, below are a few other sites you may enjoy.": "Hvala, da ste se naročili. Preden začnete brati, so tukaj še nekatere druga spletna mesta, ki vam bodo morda všeč.",
```

Comment:
AI thinks this should be nekatera - but you're the human, so feel free to say it is wrong! :)